dataservice/api/demographic/schemas.py

```diff
@@ -14,11 +14,12 @@ class DemographicSchema(BaseSchema):
                                load_only=True)
 
     class Meta(BaseSchema.Meta):
-        resource_url = 'api.demographics_list'
+        resource_url = 'api.demographics'
+        collection_url = 'api.demographics_list'
```

Comment:
It would be nice if we could figure out how to define these as part of the resource registration process.